hitboxtracker/client/src/modules/client.cpp

```diff
@@ -57,7 +57,7 @@ extra_player_info_t **CClient::FindPlayerExtraInfo()
 		return nullptr;
 	}
 
-	pos = find_pattern(pos, 32, "\x0F\xBF\x04\x2A\x2A\x2A\x2A\x2A\xC3");
+	pos = find_pattern(pos, 32, "\x0F\xBF\x04\x95\x00\x00\x00\x00\xC3");
```

Comment:
Your pattern is incorrect.
`\x00\x00\x00\x00` this not a mask, unlike `\x2A`, so this pattern will never match it.

Marked bytes should never be included in the pattern, because these will be different after reallocation on load time
![image](https://user-images.githubusercontent.com/5860435/97972650-66937700-1df7-11eb-9c57-8a2e4210ceb0.png)

Fix for latest build, just raise range search 32 up to 64